### PR TITLE
Update deploy scripts after latest constructor arg updates

### PR DESCRIPTION
**Pull request by 0xmichalis**
Update the deployment scripts to follow up on the constructor argument changes made in https://github.com/regen-network/toucan-bridge/pull/61

### CODE REVIEW

**Comment by 0xmichalis** [comment]
cc @lazaralex98

File: scripts/deploy-local.js [hunk 1]
```diff
@@ -15,18 +15,27 @@ async function deployLocal() {
...
 	console.log(`Deploying bridge contract with the following addresses:`);
 	console.log(`Owner: ${adminAccount.address}`);
 	console.log(`Token Issuer: ${bridgeAccount.address}`);
 	console.log(`NCT Pool: ${env.nctPool.address}`);
 
-	const bridge = await deployBridge(bridgeAccount.address, env.nctPool.address);
+	const bridge = await deployBridge(
+		adminAccount,
+		env.nctPool.address,
+		[DEFAULT_ADMIN_ROLE, PAUSER_ROLE, TOKEN_ISSUER_ROLE],
```
Comment (lazaralex98) [inline scripts/deploy-local.js:30]: This deploys with an admin, pauser, and token issuer, but the verification args do not have a token issuer within them. Am I missing something?

Comment (lazaralex98) [reply scripts/deploy-local.js:30]: So this is for local deployment, and that's why. I still wonder why you chose to have different args vs. the other deployment script, but this comment shouldn't be blocking anything.

**Comment by lazaralex98** [comment]
LGTM, but I can't approve.

**Pull request merged**